Projects/Windchill-App.md

```diff
@@ -0,0 +1,29 @@
+# Windchill
+
+Windchill combines the actual temperature with the wind speed to calculate
+the windchill factor, or what the perceived temperature is versus the actual
+temperature.
+
+## User Stories
+
+-   [ ] User can enter the actual temperature (Celcius) and the wind speed
+(kilometers per hour)
+-   [ ] User can press the `Calculate` button to display the wind chill
+-   [ ] User will receive and error message if data values are not entered
+or if the resulting wind chill factor is greater than or equal to the
+actual temperature.
+-   [ ] User will be prompted to enter new data values if `Calculate` is 
+pressed without first changing at least one of the input fields.
+
+## Bonus features
+
+-   [ ] User can select the measurement system calculations will be performed
```

Comment:
Other bonus feature: `User can see the wind chill factor field automatically changed when both fields have valid data` or something like that -> this means that the user doesn't have to press the button always to perform the calculation. It could be done on the `onChange` event on the inputs,

Reply:
Agreed. Good idea...great way to teach event handling.